### PR TITLE
Deploying istio update

### PR DESCRIPTION
**Pull request by tt-le**
Issue:  @[OpenLiberty/guide-istio-intro](https://github.com/OpenLiberty/guide-istio-intro/issues/106)

Updated deploy istio section to use `istioctl` for installation and update tearing down section with `istioctl`.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -118,25 +118,24 @@ include::{common-includes}/kube-start.adoc[]
 
 == Deploying Istio
 
-First, go to the https://github.com/istio/istio/releases/latest[{istio} release page^] and download the latest stable release. Extract the archive and navigate to the directory with the extracted files.
-
-Next, deploy the {istio} custom resource definitions. Custom resource definitions allow {istio} to define custom {kube} resources that you can use in your resource definition files.
+First, go to the https://github.com/istio/istio/releases/latest[{istio} release page^] and download the latest stable release. Extract the archive and navigate to the directory with the extracted files. Add `istioctl` clien to your path by running the following:
```
Comment (AustinSeto) [inline README.adoc:121]: typo "Add istioctl clien" -> "Add the istioctl client"

File: README.adoc [hunk 2]
```diff
@@ -145,9 +144,36 @@ Next, deploy {istio} resources to your cluster by running the `kubectl apply` co
...
+```
+istioctl version
+```
+
+[source, role="no_copy"]
+----
+client version: 1.7.3
+control plane version: 1.7.3
+data plane version: 1.7.3 (2 proxies)
+----
```
Comment (AustinSeto) [inline README.adoc:161]: I think this needs to go after the `istioctl install --set profile=demo` command, as I don't get this output for `istioctl version` until after I do that command.

File: README.adoc [hunk 3]
```diff
@@ -145,9 +144,36 @@ Next, deploy {istio} resources to your cluster by running the `kubectl apply` co
 
 [role=command]
 ```
-kubectl apply -f install/kubernetes/istio-demo.yaml
+export PATH=$PWD/bin:$PATH
+```
```
Comment (AustinSeto) [inline README.adoc:148]: I think this is a duplicate command box, and it still has the comment to run `kubectl apply` even though this was removed

**Review by AustinSeto — CHANGES_REQUESTED** [review]
Just a few changes

**Review by AustinSeto — CHANGES_REQUESTED** [review]
Missed these ones earlier, just some small text edits. Other than that, everything else looks good, verified on LGDEV site.

**Comment by tt-le** [comment]
@AustinSeto I just added the text edits. Thanks

File: README.adoc [hunk 4]
```diff
@@ -118,36 +118,56 @@ include::{common-includes}/kube-start.adoc[]
...
 ```
-kubectl apply -f install/kubernetes/istio-demo.yaml
+istioctl version
 ```
 
+[source, role="no_copy"]
+----
+no running Istio pods in "istio-system"
+1.7.3
+----
```
Comment (AustinSeto) [inline README.adoc:153]: Sorry missed this one too, should add a similar comment saying, "The output will be similar to the following example."

**Review by AustinSeto — CHANGES_REQUESTED** [review]

**Comment by tt-le** [comment]
@AustinSeto I just added the texts. What do you think now?

**Review by AustinSeto — APPROVED** [review]
LGTM

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@tt-le 
- **Deploying Istio**
  - [x] update the first pargaraph and remove the PATH command
- **Deploying version 1 of the system microservice**
  - [ ] "Navigate to the `start` directory ..." -> "Navigate to the `guide-istio-intro\start` directory ..." 
  - [ ] no need to have `cd start`
- **Testing microservices that are running on Kubernetes**
  - [ ] no test be run
    - need to add `mvn test-compile` before `mvn failsafe:integration-test`
    - "Run the command to start the tests:" -> "Run the following commands to compile and start the tests:"

**Comment by gkwan-ibm** [comment]
@tt-le 
- **Deploying version 1 of the system microservice**
  - change  "Navigate to the `guide-istio-intro\start` directory ..." -> "Navigate to the `guide-{projectid}\start` directory ..."

File: README.adoc [hunk 5]
```diff
@@ -116,87 +116,16 @@ include::{common-includes}/kube-start.adoc[]
...
-
+include::{common-includes}/istio-start.adoc[]
 // =================================================================================================
 // Deploying v1
 // =================================================================================================
 
 == Deploying version 1 of the system microservice
 
-Navigate to the `start` directory and run the following command to build the application locally.
+Navigate to the `guide-{projectid}\start` directory and run the following command to build the application locally.
```
Comment (gkwan-ibm) [inline README.adoc:126]: @tt-le change `guide-{projectid}\start` to `guide-{projectid}/start`

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

**Comment by tt-le** [comment]
Addressed feedback. Changed the README to use the guides-commons adoc.

**Review by Charlotte-Holt — APPROVED** [review]

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**